scripts/makefiles/Protobuf.mk

```diff
@@ -109,10 +109,22 @@ proto-storage-v1:
 		--go_out=$(PWD)/internal/storage/v1/grpc/proto/ \
 		internal/storage/v1/grpc/proto/storage_test.proto
 
+.PHONY: patch-storage-v2
+patch-storage-v2:
+	mkdir -p proto-gen/.patched/storage_v2
+	cp internal/storage/v2/grpc/trace_storage.proto proto-gen/.patched/storage_v2/
+	cp internal/storage/v2/grpc/dependency_storage.proto proto-gen/.patched/storage_v2/
```

Comment:
Why copy? It will get overwritten, is it not?

Reply:
Changed it to follow the same structure as the command for api_v3